dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
I really don't like `from_lazy_value`, and `from_dask_value` is a little unclear to me what the `value` refers too - `from_imperative` is the most clear to me. Just an opinion though, naming things is hard.